ivy.el

```diff
@@ -1963,14 +1963,14 @@ This is useful for recursive `ivy-read'."
          (let ((w (length (number-to-string
                            (length ivy--all-candidates))))
                (s (copy-sequence ivy-count-format)))
-           (string-match "%d" s)
+           (string-match-p "%d" s)
            (match-end 0)
```

Comment:
This is a no-op right? And it was in the code before your change?

Reply:
Oops, that's the issue I was talking about. This is not a no-op and my change is incorrect: before the change `string-match` modified global match data which was subsequently read by `match-end`. The `string-match-p` leaves match data intact, so `match-end` now reads whatever was previously in that global structure. I'll revert this - very unprofessional of me, sorry.

Reply:
@sergv One of the calls to `match-end` is a no-op though, as its value isn't used anywhere.

Reply:
Oh, indeed.